### DISCUSSION TITLE
Is this model used for translating sign language to text?

### DISCUSSION THREAD

**Discussion by saintlee83** [sign-language-translator/sign-language-translator#37] [Q&A]
I mean, Is this model used for translating sign language to text, not for deep learning?

**Comment by mdsrqbl** [comment]
This library is designed to provide tools to the researchers so they can easily build their own sign language translation models. The framework for rule-based translation is ready and can easily be customized. Although I will admit the documentation needs to be updated. 

The rule-based model can generate synthetic training data. In the versions >=1.0, you will be able to fine-tune our deep learning models on custom datasets.

All models will also be open source, so you can use them directly in your application as well.
